package/gluon-status-page/src/stations.c

```diff
@@ -27,6 +27,8 @@ static struct json_object *get_stations(const struct iwinfo_ops *iw, const char
     json_object_object_add(station, "signal", json_object_new_int(entry->signal));
     json_object_object_add(station, "noise", json_object_new_int(entry->noise));
     json_object_object_add(station, "inactive", json_object_new_int(entry->inactive));
+    json_object_object_add(station, "rx_rate", json_object_new_int(entry->rx_rate.rate));
+    json_object_object_add(station, "tx_rate", json_object_new_int(entry->tx_rate.rate));
```

Comment:
This change looks unrelated, and we should not add these values unless the frontend also uses them.

Reply:
in general I agree but since we are splitting backend and frontend here, I left it in the commit. I will remove it now, but I really do like those stats.